package.json

```diff
@@ -65,6 +65,7 @@
     "react-dom": "^15.3.2",
     "react-hot-loader": "^3.0.0-beta.6",
     "react-router": "^2.6.1",
+    "react-test-renderer": "^15.5.4",
```

Comment:
When using react 15.5 enzyme tries to use `react-test-renderer` and as it's not a dependency of enzyme itself it's recommended to install this alongside otherwise enzyme will warn & potentially error.

https://github.com/airbnb/enzyme/blob/75d1390147e1028301274be659dbfa7105550015/README.md#installation